src/test/scala/chiseltest/experimental/tests/VerilatorCachingTests.scala

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: Apache-2.0
+package chiseltest.experimental.tests
+
+import chisel3._
+import chiseltest.tests.{PassthroughModule, StaticModule}
+import chiseltest._
+import chiseltest.experimental.TestOptionBuilder._
+import chiseltest.internal.{CachingAnnotation, VerilatorBackendAnnotation, WriteVcdAnnotation}
+import org.scalatest._
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class VerilatorCachingTests extends AnyFlatSpec with ChiselScalatestTester with Matchers {
```

Comment:
Please add a test with multiple calls to `test` inside a single scalatest unit.

Reply:
Done, added multiple calls with the same module in one case and different modules in another case.